.github/workflows/tests.yaml

```diff
@@ -69,6 +69,10 @@ jobs:
     env:
       E2E_USERNAME:  ${{ secrets.E2E_USERNAME }}
       E2E_PASSWORD:  ${{ secrets.E2E_PASSWORD }}
+      IDP_URL: https://quantum-inspire-staging.eu.auth0.com
+      IDP_CLIENT_ID_E2E_TEST: Yz7ni9PUAyT43eUASZfmc1yqI66QxLUJ
+      COMPUTE_JOB_MANAGER_API_AUDIENCE: compute-job-manager
+      QI2_default_host: https://staging.qi2.quantum-inspire.com
```

Comment:
are we planning to keep this hardcoded here or is this supposed to be derived from secrets ?